Automattic/wp-calypso
# Stats: Unify analytics events

Related to https://github.com/Automattic/red-team/issues/42

## Proposed Changes

* add new function encapsulating event prefixing
* unify upgrade events and add parameters representing upgrade type

## Why are these changes being made?


* the changes unify the analytics process and introduce a new function that will reduce repeated code 

## Testing Instructions



* go to the live branch
* click on upgrade buttons (e.g. UTM, Devices, notices)
* verify that the new event was published alongside the old event (for data continuity)

## Pre-merge Checklist
